src/enkf/gridio_gfs.f90

```diff
@@ -355,22 +449,25 @@ subroutine readgriddata_pnc(vars3d,vars2d,n3d,n2d,levels,ndim,ntimes, &
   end if
 
   ! cloud derivatives
+  ! Currently, we do not let precipiation to affect the enkf analysis  
+  ! The following line will be removed after testing
+  use_full_hydro = .true.
```

Comment:
The comment says we do not let precipitation impact the enkf analysis but `use_full_hydro = .true.`   Should we set `use_full_hydro = .false.` as is done on line 190?

Reply:
This line is added here to ensure the changes will not affect the current result.  
It will be removed after testing.